build.sbt

```diff
@@ -265,6 +279,21 @@ lazy val testSuite = crossProject(JVMPlatform, JSPlatform)
     libraryDependencies += "io.github.cquiroz" %%% "cldr-api" % cldrApiVersion
   )
   .jvmConfigure(_.dependsOn(macroUtils))
+  .nativeSettings(
+    commonNativeSettings,
+    nativeConfig ~= {
+      _.withOptimize(false)
+      // tests fail to link on Scala 2.11 and 2.12 in debug mode
```

Comment:
Anyway, it's a Scala Native bug that we can't circumvent.
I reported it in the Discord channel and they will look at it. Once a new version of Scala Native will solve this it will be solved for scala-java-locales users automatically (by my best guess, since it happens in linking).
So I don't think we should worry about it.